test/fixtures/vcr_cassettes/get_users.json

```diff
@@ -13,7 +13,7 @@
2d7aeea2c2\", \"id\": \"user_mZTG4UEHZCYqcg2BXaaLts6bPDIOSuWlvLgRMoV3zrN\", \"last_used_timezone\": \"Europe/London\", \"date_created\": \"2017-04-24T08:40:42.138000+00:00\", \"email\": \"henry@nested.com\"}, {\"organizations\": [\"orga_qmaQhybBvwhwFbysDwnkwDGzUJfFr8Mt82UZUFArRcP\"], \"first_name\": \"Jai\", \"last_name\": \"Khurmi\", \"date_updated\": \"2017-12-18T03:15:14.521000+00:00\", \"image\": \"https://secure.gravatar.com/avatar/957017139816b764dd0935344bbfef0f\", \"id\": \"user_pPxDCqtqWDWiTGELqSel20Ts3XyLFEuIlW04fRLxzAA\", \"last_used_timezone\": \"Europe/London\", \"date_created\": \"2017-07-25T17:02:50.227000+00:00\", \"email\": \"jai@nested.com\"}, {\"organizations\": [\"orga_qmaQhybBvwhwFbysDwnkwDGzUJfFr8Mt82UZUFArRcP\"], \"first_name\": \"Susan\", \"last_name\": \"Gregory\", \"date_updated\": \"2017-12-18T03:15:45.691000+00:00\", \"image\": \"https://secure.gravatar.com/avatar/e9520694e273f8ed41342b9e147da419\", \"id\": \"user_s9370UPzs7ShaPjq4VMZsG7vovxrbLRAoIE7Y2feuVP\", \"last_used_timezone\": \"Europe/London\", \"date_created\": \"2017-06-09T09:32:24.025000+00:00\", \"email\": \"susan@nested.com\"}, {\"organizations\": [\"orga_qmaQhybBvwhwFbysDwnkwDGzUJfFr8Mt82UZUFArRcP\"], \"first_name\": \"Freddie\", \"last_name\": \"Elliott\", \"date_updated\": \"2017-12-18T03:15:14.442000+00:00\", \"image\": \"https://secure.gravatar.com/avatar/8085a0ecaa2b7c17d2aa3df2f2af2b0e\", \"id\": \"user_t8xhNhdcAeBdAUiJaqhpaJuN6PuOSN4zaCcrlaea71u\", \"last_used_timezone\": \"Europe/London\", \"date_created\": \"2017-10-30T09:25:59.199000+00:00\", \"email\": \"freddie@nested.com\"}, {\"organizations\": [\"orga_qmaQhybBvwhwFbysDwnkwDGzUJfFr8Mt82UZUFArRcP\"], \"first_name\": \"Bob\", \"last_name\": \"Crowley\", \"date_updated\": \"2017-12-18T03:16:15.277000+00:00\", \"image\": \"https://secure.gravatar.com/avatar/e0e3d90e70d3e578956e833528d6e487\", \"id\": \"user_tGK8Xp5Fl6LpcFfAWXWdKkFg2PzgVWEv8I96texckYz\", \"last_used_timezone\": \"Europe/London\", \"date_created\": \"2017-09-18T10:25:41.547000+00:00\", \"email\": \"bob@nested.com\"}, {\"organizations\": [\"orga_qmaQhybBvwhwFbysDwnkwDGzUJfFr8Mt82UZUFArRcP\"], \"first_name\": \"Andy\", \"last_name\": \"Harris\", \"date_updated\": \"2017-12-18T03:15:04.656000+00:00\", \"image\": \"https://secure.gravatar.com/avatar/f0638325de8165e27b474d9f9a119634\", \"id\": \"user_v9yUukdKu0f6dUE0lorzRWrkwMebLBZbjpvOzkKtNma\", \"last_used_timezone\": \"Europe/London\", \"date_created\": \"2017-08-29T13:48:36.306000+00:00\", \"email\": \"andy@nested.com\"}, {\"organizations\": [\"orga_qmaQhybBvwhwFbysDwnkwDGzUJfFr8Mt82UZUFArRcP\"], \"first_name\": \"Bonnie\", \"last_name\": \"Pooley\", \"date_updated\": \"2017-12-18T03:16:19.179000+00:00\", \"image\": \"https://secure.gravatar.com/avatar/9bdce6c2f30296f80edf18b80c642d51\", \"id\": \"user_xdG0HLQezJDX2ALlcFevERSCv2PeEc3tbipa308oVnn\", \"last_used_timezone\": \"Europe/London\", \"date_created\": \"2017-12-13T15:30:52.002000+00:00\", \"email\": \"bonnie@nested.com\"}]}",
```

Comment:
Not a nice diff. I've changed him to "Mister Blobby"

Reply:
![image](https://user-images.githubusercontent.com/53174838/199029981-ce8d1ea9-1006-4798-81e1-c15ad800d5f7.png)